core/src/main/java/com/novoda/espresso/TalkBackTestRule.java

```diff
@@ -0,0 +1,42 @@
+package com.novoda.espresso;
+
+import android.os.SystemClock;
+
+import org.junit.rules.TestRule;
+import org.junit.runner.Description;
+import org.junit.runners.model.Statement;
+
+public class TalkBackTestRule implements TestRule {
+
+    @Override
+    public Statement apply(Statement base, Description description) {
+        return new TalkBackStatement(base);
+    }
+
+    private static class TalkBackStatement extends Statement {
+
+        private static final int DEFAULT_DELAY_MILLIS = 1500;
+
+        private final TalkBackStateSettingRequester talkBackStateSettingRequester = new TalkBackStateSettingRequester(0);
```

Comment:
doing the sleep in the requester seems not to prevent the `baseStatement.evaluate()` from being called such that the activity is started before talkback has been turned on.

Instead, moving the sleep to this class ensures we don't start the activity without giving a chance for talkback to initialize.